src/commands/signzone.rs

```diff
@@ -735,10 +753,9 @@ impl SignZone {
 
                 // Verify that the owner of the public key matches the apex of the
                 // zone.
-                if public_key.owner() != soa_rr.owner() {
+                if public_key.owner() != apex {
                     return Err(format!(
-                        "Zone apex ({}) does not match the expected apex ({})",
-                        soa_rr.owner(),
+                        "Zone apex ({apex}) does not match the expected apex ({})",
```

Comment:
Should this say something like 'public key owner name does not match zone apex'?

Reply:
Yes. I'll fix it.